contrib/tablefunc/tablefunc--1.0--1.1.sql

```diff
@@ -0,0 +1,16 @@
+/* contrib/tablefunc/tablefunc--1.0--1.1.sql */
+
+-- complain if script is sourced in psql, rather than via ALTER EXTENSION
+\echo Use "ALTER EXTENSION tablefunc UPDATE TO '1.1'" to load this file. \quit
+
+ALTER function normal_rand(integer,double precision,double precision) EXECUTE ON INITPLAN;
```

Comment:
Do we really need to add normal_rand on initplan? Can you show any examples with reproducing the problem. Seems like it works fine in old code. E.g.
```sql
create table t_avg as select * from normal_rand(5, 250, 0);
```